cmd/zetaclientd/start.go

```diff
@@ -238,11 +238,14 @@ func start(_ *cobra.Command, _ []string) error {
 
 	// stop zetacore observer
 	for _, chain := range cfg.GetEnabledChains() {
+		_, ok := chainClientMap[chain]
```

Comment:
Maybe we could instead iterate the map instead of calling `cfg.GetEnabledChains()`?

Reply:
> Maybe we could instead iterate the map instead of calling `cfg.GetEnabledChains()`?

Sounds a good idea.